### PR TITLE
Remove unwanted objects from validator.ts

### PR DESCRIPTION
**Pull request by xydas97**
Remove Parameters and ParameterFields from validator.ts

### CODE REVIEW

**Comment by patrickkuo** [comment]
https://github.com/MystenLabs/sui/pull/8848 contain some of the same changes already

File: crates/sui-types/src/sui_system_state/sui_system_state_inner_v1.rs [hunk 1]
```diff
@@ -31,7 +31,7 @@ const E_METADATA_INVALID_WORKER_ADDR: u64 = 7;
 /// Rust version of the Move sui::sui_system::SystemParameters type
 #[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
 // TODO: Get rid of json schema once we deprecate getSuiSystemState RPC API.
-#[serde(rename = "SystemParameters")]
+#[serde(rename = "SystemParameters", rename_all = "camelCase")]
```
Comment (666lcz) [inline crates/sui-types/src/sui_system_state/sui_system_state_inner_v1.rs:34]: `getSuiSystemState` has been removed. Could you help removing all these?

File: crates/sui-types/src/governance.rs [hunk 2]
```diff
@@ -28,6 +28,7 @@ pub const ADD_DELEGATION_LOCKED_COIN_FUN_NAME: &IdentStr =
 pub const WITHDRAW_DELEGATION_FUN_NAME: &IdentStr = ident_str!("request_withdraw_delegation");
 
 #[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
```
Comment (666lcz) [inline crates/sui-types/src/governance.rs:31]: Good catch!

Comment (patrickkuo) [reply crates/sui-types/src/governance.rs:31]: actually we no longer expose StakedSui via RPC, we can remove the serde tag

**Review by 666lcz — APPROVED** [review]

**Pull request merged**